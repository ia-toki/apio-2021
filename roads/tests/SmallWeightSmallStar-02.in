c10234cabdfd6eae47773fb6f163e0350fc797e1
200
0 40 10
0 17 9
0 71 8
0 61 3
0 126 5
0 47 1
0 64 8
0 128 6
0 25 2
0 168 5
0 196 1
0 75 3
0 177 9
0 190 8
0 197 3
0 97 10
0 19 2
0 187 8
0 48 1
0 4 9
0 154 4
0 94 5
0 141 6
0 68 8
0 143 5
0 5 8
0 172 2
0 155 1
0 112 7
0 72 1
0 33 8
0 127 8
0 101 9
0 108 3
0 9 3
0 73 9
0 84 5
0 153 3
0 87 10
0 63 3
0 160 4
0 32 4
0 10 10
0 175 5
0 169 2
0 82 4
0 29 9
0 162 4
0 167 6
0 192 4
0 45 2
0 150 8
0 62 1
0 185 5
0 31 4
0 58 8
0 194 2
0 60 9
0 156 3
0 26 10
0 89 2
0 39 10
0 28 2
0 189 4
0 145 10
0 129 1
0 104 9
0 67 1
0 74 7
0 77 8
0 164 8
0 176 1
0 140 4
0 85 9
0 124 5
0 86 10
0 50 8
0 69 5
0 149 5
0 37 5
0 139 4
0 118 4
0 79 7
0 23 1
0 103 6
0 121 3
0 136 3
0 55 8
0 147 1
0 180 3
0 16 1
0 119 10
0 27 5
0 134 8
0 144 9
0 91 2
0 178 8
0 174 2
0 110 10
0 80 7
0 195 10
0 107 10
0 138 9
0 30 7
0 188 6
0 11 7
0 59 3
0 83 3
0 100 3
0 54 9
0 98 8
0 93 5
0 186 8
0 158 3
0 157 9
0 66 10
0 142 9
0 51 4
0 92 8
0 198 9
0 193 3
0 8 9
0 2 9
0 183 10
0 117 9
0 15 1
0 7 6
0 120 8
0 90 5
0 20 7
0 96 7
0 52 8
0 65 6
0 199 6
0 57 5
0 6 9
0 18 7
0 22 3
0 125 5
0 115 10
0 46 4
0 111 7
0 3 2
0 166 7
0 34 1
0 102 2
0 170 5
0 24 4
0 159 3
0 113 10
0 44 6
0 191 1
0 70 5
0 146 9
0 179 9
0 165 4
0 163 6
0 35 3
0 184 8
0 131 1
0 53 8
0 106 2
0 135 5
0 42 5
0 171 4
0 123 10
0 56 5
0 95 1
0 76 7
0 43 3
0 99 8
0 49 6
0 181 8
0 182 3
0 148 2
0 78 3
0 161 4
0 137 4
0 133 5
0 130 5
0 81 5
0 21 4
0 132 5
0 14 1
0 13 3
0 38 3
0 109 3
0 114 9
0 1 3
0 122 8
0 151 5
0 105 7
0 36 8
0 173 10
0 152 2
0 88 2
0 41 7
0 116 8
0 12 10
