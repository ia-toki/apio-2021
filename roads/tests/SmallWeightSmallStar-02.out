064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
1110 1100 1090 1080 1070 1060 1050 1040 1030 1020 1010 1000 990 980 970 960 950 940 930 920 911 902 893 884 875 866 857 848 839 830 821 812 803 794 785 776 767 758 749 740 731 722 714 706 698 690 682 674 666 658 650 642 634 626 618 610 602 594 586 578 570 562 554 546 538 530 522 514 506 498 491 484 477 470 463 456 449 442 435 428 421 414 407 400 394 388 382 376 370 364 358 352 346 340 334 329 324 319 314 309 304 299 294 289 284 279 274 269 264 259 254 249 244 239 234 229 224 219 214 209 204 200 196 192 188 184 180 176 172 168 164 160 156 152 148 144 140 136 132 128 125 122 119 116 113 110 107 104 101 98 95 92 89 86 83 80 77 74 71 68 65 62 59 56 53 50 48 46 44 42 40 38 36 34 32 30 28 26 24 22 20 18 17 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0
